### PR TITLE
doc pass 12 flashlights

### PR DESCRIPTION
**Pull request by alganthe**

### CODE REVIEW

File: wiki/feature/flashlights.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+group: feature
+category: realism
+parent: wiki
+---
+
+## 1. Overview
+Adds handheld flashlights that will give you the ability to read your map even in the darkest environments.
+
+### 1.1 A pvp friendly item
+When using those flashlights you will have a slight glow around you making this feature pvp friendly.
```
Comment (bux578) [inline wiki/feature/flashlights.md]: PvP

File: wiki/feature/flashlights.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## 2. Usage
+
+### 2.1 Turning the flashlights on/off
+- For this you obviously need one of the flashlights (`Fulton MX-991`,`Maglite XL50`,`KSF-1`).
```
Comment (bux578) [inline wiki/feature/flashlights.md]: ~~obviously~~

File: wiki/feature/flashlights.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- On the map screen self interact <kbd>Ctrl</kbd>+<kbd>⊞&nbsp;Win</kbd>.
```
Comment (bux578) [inline wiki/feature/flashlights.md]: ... map screen use self interact ...

File: wiki/feature/flashlights.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- Select `Flashlights`.
+- Select the flashlight you want to use and select `on`.
```
Comment (bux578) [inline wiki/feature/flashlights.md]: `On`(?)

File: wiki/feature/flashlights.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+- Same process for turning it off.
```
Comment (bux578) [inline wiki/feature/flashlights.md]: (Select the flashlight you want to use and select `on`.)?

**Comment by voiperr** [comment]
It should be mentioned that the flashlight state is persistent: if you turn a flashlight on, close the map, and then reopen it, the flashlight will be on again.

**Pull request merged**